src/docs/Components/Buttons/index.mdx

```diff
@@ -11,80 +11,80 @@ status:
 
 ## Variations
 
-### Four color schemes for each type of action
+### Color schemes
```

Comment:
In the code, we don't have **styles** and **schemes**, we have **themes** and they are implemented like this : 
`primary` (default), `bordered`, `solid-primary-02`, `bordered-primary-02`, `solid-danger`, `bordered-danger`, `solid-neutral`, `bordered-neutral`

And we don't have a choice, because of the customization.
We need to have the same structure in the usage docs.

Reply:
we could say:
Color themes

We created different color themes to suit what you need in your design. You can use `primary` (default), `danger` and `neutral` theme in `solid` and `bordered` declination.

Reply:
we could say:
Color themes

We created different color themes to suit what you need in your design. You can use `primary` (default), `danger` and `neutral` theme in `solid` and `bordered` declination.

Reply:
looks good to me :)

Reply:
fixed

Reply:
not solved

Reply:
I let the designers handle this point.
@caqueste @nherwegh Can you take a look at it?

Reply:
![image](https://user-images.githubusercontent.com/27195157/70543486-bd062000-1b6a-11ea-8dc5-8ba53d0acf5c.png)
seems solved for me